scripts/install_local.sh

```diff
@@ -22,7 +22,9 @@ do
     then
         kubectl replace --force -f "${f}"
     else
-        kubectl apply -f "${f}"
+	if ! kubectl create -f "${f}"; then
+		kubectl replace -f "${f}"
+	fi
```

Comment:
Any value in adding a comment to explain why this needed?

Reply:
sure. I can add a comment